### PR TITLE
Updates to address issue #355

### PR DESCRIPTION
**Pull request by taquitos**
Fixes for #355

### CODE REVIEW

File: features/project/project_controller.rb [hunk 1]
```diff
@@ -73,15 +79,18 @@ class ProjectController < AuthenticatedControllerBase
       # So a new project is created with default settings so we can fetch it.
       repo_config = GitRepoConfig.from_octokit_repo!(repo: selected_repo)
 
+      require "tmpdir"
```
Comment (KrauseFx) [inline features/project/project_controller.rb]: Maybe move this to the other `require`, unless this is for performance reasons

File: features/project/project_controller.rb [hunk 2]
```diff
@@ -121,12 +130,15 @@ class ProjectController < AuthenticatedControllerBase
       lane = params["selected_lane"]
       project_name = params["project_name"]
 
+      require "tmpdir"
```
Comment (KrauseFx) [inline features/project/project_controller.rb]: Same here

File: features/project/project_controller.rb [hunk 3]
```diff
@@ -160,14 +172,16 @@ class ProjectController < AuthenticatedControllerBase
 
       relative_fastfile_path = nil
       available_lanes = []
-      absolute_fastfile_path = project.local_fastfile_path
-      unless absolute_fastfile_path.nil?
-        parser = Fastlane::FastfileParser.new(path: absolute_fastfile_path)
-        available_lanes = parser.available_lanes
+      # TODO: Follow up in another PR
```
Comment (KrauseFx) [inline features/project/project_controller.rb]: Let's create an issue for this, and link is, so we it's easy for people to grab

File: services/services.rb [hunk 4]
```diff
@@ -63,14 +64,16 @@ def self.ci_config_repo
         hidden: true
       )
     end
+    #        containing_path: File.expand_path("~/.fastlane/ci/"),
```
Comment (KrauseFx) [inline services/services.rb]: ⬆️

File: features/project/project_controller.rb (hunk 3 above)
Comment (taquitos) [reply features/project/project_controller.rb]: yup, was going to do that after merge 👍

**Comment by KrauseFx** [comment]
awesome 💯

File: features/project/project_controller.rb (hunk 3 above)
Comment (taquitos) [reply features/project/project_controller.rb]: https://github.com/fastlane/ci/issues/357

**Pull request merged**